### DISCUSSION TITLE
Running Cross-Platform App on Linux

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#9330] [Q&A] [answered]
In v11, I created a new project template on Linux. Trying to run it.

Desktop version works and now respects the dark theme out-of-the-box.

Blazor version throws `System.DllNotFoundException: libSkiaSharp`

Android version cannot compile, `Android` namespace not recognized.

iOS version cannot compile, `UIKit` namespace not recognized.

Can these be run on Linux?

**Comment by mysteryx93** [comment, marked as answer]
Found this for Android / iOS

https://docs.avaloniaui.net/tutorials/developing-for-mobile/ios/setting-up-your-developer-environment-for-ios
https://docs.avaloniaui.net/tutorials/developing-for-mobile/android/setting-up-your-developer-environment-for-android

I'll need to update .NET SDK first though... been having issues with Arch system updates lately.

**Comment by maxkatz6** [comment]
FYI, you won't be able to run iOS app on linux. And on Windows it's possible only with proxying build through real macOS device.

About linux app: https://github.com/AvaloniaUI/Avalonia/issues/5190#issuecomment-748724948

**Comment by timunie** [comment]
Converted this into Q&A as this is neither a bug nor a feature request. If you disagree please let us know.
